src/App.js

```diff
@@ -74,8 +78,15 @@ class App extends Component {
     })
   }
 
+  deleteMap(mapId) {
+    return Api.deleteMap(mapId).then(() => {
+      this.loadMaps()
+    })
+  }
+
   setSelectedMap(mapYear) {
     const map = this.state.maps.find(map => map.year === mapYear)
```

Comment:
This is where we can use the new year endpoint. We will then not need to find the map since we will already have that info readily available